test/elm/interval/data.js

```diff
@@ -102774,15 +102774,35 @@ define CollapseQuantityIntervalListWithNullHighNoOverlap: collapse QuantityInter
 define CollapseQuantityIntervalListWithNullHighNoOverlapExpected: { Interval[ToQuantity(1), ToQuantity(2)], Interval[ToQuantity(4), null]}
 */
 
+/*
+Translation Error(s):
+[71:22, 71:34] Call to operator Collapse(System.Any,System.Quantity) is ambiguous with: 
+  - Collapse(list<interval<System.Integer>>,System.Quantity)
+  - Collapse(list<interval<System.Quantity>>,System.Quantity)
+  - Collapse(list<interval<System.Date>>,System.Quantity)
+  - Collapse(list<interval<System.DateTime>>,System.Quantity)
+  - Collapse(list<interval<System.Time>>,System.Quantity)
+*/
 module.exports['Collapse'] = {
    "library" : {
       "annotation" : [ {
          "translatorOptions" : "EnableDateRangeOptimization,EnableAnnotations",
          "type" : "CqlToElmInfo"
+      }, {
+         "libraryId" : "TestSnippet",
+         "libraryVersion" : "1",
+         "startLine" : 71,
+         "startChar" : 22,
+         "endLine" : 71,
+         "endChar" : 34,
+         "message" : "Call to operator Collapse(System.Any,System.Quantity) is ambiguous with: \n  - Collapse(list<interval<System.Integer>>,System.Quantity)\n  - Collapse(list<interval<System.Quantity>>,System.Quantity)\n  - Collapse(list<interval<System.Date>>,System.Quantity)\n  - Collapse(list<interval<System.DateTime>>,System.Quantity)\n  - Collapse(list<interval<System.Time>>,System.Quantity)",
+         "errorType" : "semantic",
+         "errorSeverity" : "error",
+         "type" : "CqlToElmError"
```

Comment:
Note that the Translation error is still here -- so the output ELM is not what you think it is.